docs/docsite/rst/network/getting_started/network_roles.rst

```diff
@@ -0,0 +1,37 @@
+
+.. _using_network_roles:
+
+******************************************
+Use Ansible network roles
+******************************************
+
+The Ansible Network team develops and supports a set of `network-related roles <https://galaxy.ansible.com/ansible-network>`_ on Galaxy. You can use these roles to jumpstart your network automation efforts. These roles are updated approximately every two weeks to give you access to the latest Ansible networking content.
+
+These roles come in the following categories:
+
+* **User roles** - You use these roles, such as `config_manager <https://galaxy.ansible.com/ansible-network/config_manager>`_ and `cloud_vpn <https://galaxy.ansible.com/ansible-network/cloud_vpn>`_, directly in your playbooks. These roles are platform/provider agnostic, allowing you to use the same roles and playbooks across different network platforms or cloud providers.
```

Comment:
I'd add a sentence at the beginning here to describe what user roles do - something like "User roles focus on tasks, such as managing configuration or ????." Then add the part about using these roles directly in playbooks. I'd also add an example (either here, or elsewhere with a link from here).